server/controllers/complaintHandler.js

```diff
@@ -0,0 +1,118 @@
+const DocComplaint = require('../database/models/docComplaint');
+const UserComplaint = require('../database/models/userComplaint');
+const authentication = ('../authentication');
```

Comment:
require()
Fix this in [docHandlers](https://github.com/chc5/gitgud/blob/master/server/controllers/docHandlers.js) and [testing](https://github.com/chc5/gitgud/blob/master/server/controllers/testing.js) as well :)